frontend/src/css/style.css

```
@@ -1,59 +1,68 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 .fc {
-  font-family: 'IBM Plex Sans', system-ui, sans-serif;
+  font-family: 'Inter', system-ui, sans-serif;
   --fc-border-color: var(--border-subtle);
   --fc-page-bg-color: var(--bg-card);
   --fc-today-bg-color: transparent;
   --fc-highlight-color: transparent;
+  touch-action: none;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)